src/connectors/xiami.js

```diff
@@ -1,22 +1,17 @@
 'use strict';
 
-Connector.playerSelector = '#player-main';
+Connector.playerSelector = '.play-bar';
 
-Connector.trackSelector = '#J_trackName';
+Connector.trackSelector = '.music .info .title';
 
-Connector.albumSelector = '.ui-track-current .c3>a';
+Connector.artistSelector = '.music .info .singers';
 
-Connector.artistSelector = '#J_trackInfo>a:nth-child(2)';
+Connector.playButtonSelector = '.main-control .play-btn';
 
-Connector.playButtonSelector = '.play-btn';
+Connector.currentTimeSelector = '.audio-progress .range .bar .handle';
 
-Connector.currentTimeSelector = '#J_positionTime';
+Connector.trackArtSelector = '.music .cover-link .active img';
 
-Connector.trackArtSelector = '#J_playerCoverImg';
-
-Connector.getDuration = () => {
-	let text = Connector.getCurrentTime() ? $('#J_durationTime').text() : '';
-	return Util.stringToSeconds(text);
+Connector.isPlaying = () => {
```

Comment:
Do you need this? Would it work with just the defined `playButtonSelector`?

Reply:
It doesn't work well without an isPlaying function.

Reply:
Can we drop the `playButtonSelector` definition then?

Reply:
I erased the playButtonSeltter and tested it, but there was a service error.

Reply:
Really? Weird. this can be used instead of the isPlaying function.

Reply:
I don't know why this is happening, but it doesn't work properly without either `isPlaying` or `playButtonSeltter` method.

Reply:
I just drop the `playButtonSector` and tested it and it works. I'll make a new commitment and post it. 👍